manifests/server.pp

```diff
@@ -268,6 +271,7 @@
   Optional[String[1]] $zabbix_user                                            = $zabbix::params::server_zabbix_user,
   Boolean $manage_startup_script                                              = $zabbix::params::manage_startup_script,
   Optional[Stdlib::Absolutepath] $socketdir                                   = $zabbix::params::server_socketdir,
+  Optional[String] $webserviceurl                                             = undef,
```

Comment:
instead of String, we could use something from stdlib I guess, for example https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/httpsurl.pp

Reply:
sounds good, let me test.